src/args.rs

```diff
@@ -1,8 +1,14 @@
-#[derive(Debug)]
-pub struct Args {}
+use clap::Parser;
 
-impl Args {
-    pub fn parse() -> Self {
-        todo!()
-    }
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+pub struct Args {
+	/// Application name
+	#[arg(long)]
+	app: String,
+	
+	/// Most recent time interval in milliseconds
+	#[arg(short, long, default_value_t = 1000)]
+	interval: u32,
 }
+
```

Comment:
It would be nice to add a few getters and some tests here just to make sure our API definitely works, something like the following would be good
```rust
impl Args {
    fn app(&self) -> &str {
        &self.app
    }

    ...
}

#[cfg(test)]
mod test {
    use super::*;

    #[test]
    fn app() {
        assert!(Args::try_parse(["jperf"]).is_err()); // (this just tests that app is mandatory, delete this comment)
        assert_eq!("foo", Args::try_parse(["jperf", "--app", "foo"]).unwrap().app());
    }

    ...
}
```

Reply:
Noice! I'll add these